### ISSUE TITLE
Create WorkflowConfig object to re-use across API calls

### ISSUE THREAD

**Issue by PatStLouis** [w3c-ccg/vc-api#413]
Following #409, we want to create a workflowConfig object to re-use across calls

**Comment by PatStLouis** [comment]
Point of discussion, the `steps` property.

The model currently looks like this:
```
steps [object]
    One or more steps required to complete an exchange on the workflow. The steps object MUST be an object of the following form:

    stepName [object]
        The name of the step. The stepName object MUST be an object of the following form:

        step [object]
            Object containing information about a workflow step. The step object MUST be either an object of the following form: 
```

While I understand what it's meant to represent, the way it's modeled is confusing such as the description of the `stepName` being the name of the step while being an object.

**Comment by msporny** [comment]
The group discussed this on the 2024-09-10 call:

@PatStLouis noted his comment above as a concern. @dlongley noted that "steps" is an object where they keys are the name of the steps. Lots of discussion, but end result was to modify the text and possibly the code to ensure that the output is easier to read.

**Comment by kezike** [comment]
As I was reviewing the latest version of the [workflow creation](https://w3c-ccg.github.io/vc-api/#create-workflow) and [workflow retrieval](https://w3c-ccg.github.io/vc-api/#get-workflow-configuration) endpoints, I noticed that there is an intermediate `step` property. This seems unnecessary and may be error prone for implementers. Could we remove that?

**Comment by dlongley** [comment]
@kezike,

Yes, the "intermediate `step` property" you mention is a bug somewhere in the yaml or in how it's processed. It will be removed. The "steps" object has keys that are the step names and values that are the step objects. There should be no intermediate `step` property at all.

**Comment by BigBlueHat** [comment]
Closed via #494
